po/zh_CN.po

```diff
@@ -78933,6 +78955,9 @@ msgstr "缩放"
 msgid "~ expansion"
 msgstr "~ 扩大"
 
+#~ msgid "Could not show help message"
+#~ msgstr "无法显示帮助消息"
```

Comment:
can we make it translatable first, so it's not removed in this commit?
I know it's just one string but that would be a proof that things work.